app/controllers/document_images_controller.rb

```diff
@@ -119,6 +119,8 @@ def delete_image_from_asset_manager(image)
   end
 
   def update_crop_params
-    params.permit(:crop_x, :crop_y, :crop_width, :crop_height)
+    image_aspect_ratio = Image::HEIGHT.to_f / Image::WIDTH
+    crop_height = params[:crop_width].to_i * image_aspect_ratio
```

Comment:
Are the images always horizontal? would they ever be verticle?

Reply:
I'm not sure what you mean here - could you elaborate? The image constants imply a horizontal image orientation, if that helps.